franklin/builder/serializers.py

```diff
@@ -0,0 +1,54 @@
+import os
+
+from rest_framework import serializers
+
+from builder.models import Owner, Site
+
+
+class OwnerSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Owner
+        fields = ('name', 'github_id')
+        extra_kwargs = {
+            "github_id": {
+                "validators": [],
+            },
+        }
+
+class SiteSerializer(serializers.ModelSerializer):
+    owner = OwnerSerializer()
+    
+    class Meta:
+        model = Site
+        fields = ('name', 'github_id', 'owner')
+        # Not ideal, but you can't update existing models without disabling
+        # validation for our unique=True github_id
+        # This is a known issue in DRF:
+        #   https://github.com/tomchristie/django-rest-framework/issues/2996
+        #   https://groups.google.com/d/msg/django-rest-framework/W6F9_IJiZrY/6gDa_GmhcasJ
+        # If this is fixed, add update() and rewrite create() to be more like:
+        #   Site.objects.create(owner=owner, **validated_data)
+        extra_kwargs = {
+            "github_id": {
+                "validators": [],
+            },
+        }
```

Comment:
DRF seems to be aware that modelSerializers are broken. Though they don't seem crazy eager to fix it. Ticket status is currently "needs design decision". Hopefully that just means they are going to take it slow and implement a beautiful solution.